weather/csv_import.py

```diff
@@ -0,0 +1,50 @@
+###################
+#csv_import.py
```

Comment:
Can we add a description to this header about where the historical files are downloaded from if someone were to reuse this script?